src/main/scala/com/fulcrumgenomics/util/SampleBarcodeMetric.scala

```diff
@@ -43,11 +43,15 @@ object SampleBarcodeMetric {
     var totalReads: Long           = 0
     var totalPfReads: Long         = 0
     var totalPfReadsAssigned: Long = 0
+    var totalBases: Long           = 0
+    var totalBasesPassQ30: Long    = 0
```

Comment:
```suggestion
    var totalBasesPass: Long    = 0
```

Reply:
see this comment: https://github.com/fulcrumgenomics/fgbio/pull/720#issuecomment-932328730